va/va.h

```diff
@@ -492,6 +492,20 @@ typedef enum
      */
     VAConfigAttribEncMacroblockInfo     = 16,
     /**
+     * \brief Maximum picture width. Read-only.
+     *
+     * This attribute determines the maximum picture width the driver supports
+     * for a given configuration.
+     */
+    VAConfigAttribMaxPictureWidth     = 18,
+    /**
+     * \brief Maximum picture height. Read-only.
+     *
+     * This attribute determines the maximum picture height the driver supports
+     * for a given configuration.
+     */
+    VAConfigAttribMaxPictureHeight    = 19,
+    /**
```

Comment:
This should be two different things, one is for codec capability (each codec would be different), one is for generic surface capability. A larger surface can be used for smaller pictures. BTW, is vaQuerySurfaceAttributes() required before creating the surface?